src/core_plugins/metrics/public/components/lib/get_axis_label_string.js

```diff
@@ -0,0 +1,25 @@
+import { relativeOptions } from '../../../../../ui/public/timepicker/relative_options';
+import _ from 'lodash';
+import moment from 'moment';
+const unitLookup = {
+  s: 'seconds',
+  m: 'minutes',
+  h: 'hours',
+  d: 'days',
+  w: 'weeks',
+  M: 'months',
+  y: 'years'
+};
+export function getAxisLabelString(interval) {
+  const units = _.pluck(_.clone(relativeOptions).reverse(), 'value')
```

Comment:
The painful part of trying to use `time_units.js` is that it's setting a constant for Angular and the code that uses this is in React.